herddb-core/src/test/java/herddb/codec/RecordSerializerTest.java

```diff
@@ -47,7 +49,7 @@ public RecordSerializerTest() {
     public void testToBean() {
         Table table = Table.builder()
             .name("t1")
-            .column("pk", ColumnTypes.STRING)
+            .column("pk", ColumnTypes.NOTNULL_STRING)
```

Comment:
do not change this test, add a new one if you think it has some value

Reply:
👍